src/email/index.ts

```diff
@@ -28,7 +28,7 @@ export const handler = async (event: AppSyncEvent): Promise<any> => {
     switch (fieldName) {
       case 'createSendEmail':
         {
-          const { senderEmail, receiverEmail, body, subject, mailingList } = args;
+          const { senderEmail, receiverEmail, body, subject, mailingList, sendIndividual } = args;
```

Comment:
@SonuKumar81800 why dont you make sendEmail and sendBulkEmail to independent function?